src/commands/up.rs

```diff
@@ -41,6 +42,7 @@ pub struct UpCommand {
             long = "bindle",
             conflicts_with = APP_CONFIG_FILE_OPT,
             requires = BINDLE_SERVER_URL_OPT,
+            hide = true,
```

Comment:
Again, probably only a deprecation message at the point?

Reply:
Punting bindle deprecation out of this PR.